libs/langchain/langchain/memory/chat_message_histories/streamlit.py

```diff
@@ -0,0 +1,36 @@
+from typing import List
+
+from langchain.schema import (
+    BaseChatMessageHistory,
+)
+from langchain.schema.messages import BaseMessage
+
+
+class StreamlitChatMessageHistory(BaseChatMessageHistory):
+    """
+    Chat message history that stores messages in Streamlit session state.
+
+    Args:
+        message_key: session state key to use for storing messages.
+    """
+
+    def __init__(self, message_key: str = "messages"):
```

Comment:
super nit: `message_key` sounds a bit misleading, maybe `messages_key`, `memory_key`, `state_key`, or just `key` since every Streamlit user probably knows this concept from widgets.

Reply:
Good call, changed to `key`